test/sequential/sequencerStaking.cancel.test.ts

```diff
@@ -363,12 +363,20 @@ it("GIVEN a sequencer, WHEN <in-correctly> canceling an update AND cancelator pr
         .on(chain)
         .build(),
     ),
-    Sudo.sudoAs(testUser2, providingExtrinsic),
+    Sudo.sudoAs(
+      testUser2,
+      await SequencerStaking.provideSequencerStaking(
+        (await SequencerStaking.minimalStakeAmount()).muln(2),
+        chain,
+        "StakeOnly",
+      ),
+    ),
   );
   await testUser2.refreshAmounts(AssetWallet.BEFORE);
   await waitSudoOperationSuccess(cancelResolutionEvents, "SudoAsDone");
   await waitForNBlocks(disputePeriodLength + 1);
   await testUser2.refreshAmounts(AssetWallet.AFTER);
+  await Sudo.asSudoFinalized(Sudo.sudoAs(testUser2, providingExtrinsic));
```

Comment:
why the user needs to rejoin? 
 + rename the "providingExtrinsic" by "stakeAndJoinExtrinsic" to match the real behavior.

Reply:
It's my mistake, deleted. Variable was renamed